pkg/mesh-networking/translation/istio/mesh/access/access_policy_translator.go

```
@@ -162,7 +166,7 @@ func ingressGatewayAuthPolicyName(ingressGateway *commonv1.AppliedIngressGateway
 
 	hash, err := hashutils.HashAllSafe(nil, name)
 	if err != nil {
-		// TODO(ryantking): Handle hashing error
+		contextutils.LoggerFrom(ctx).DPanicw("unable to hash name", zap.String("name", name), zap.Error(err))
```

Need to also return this error, wrapped with a helpful note